website/docs/tutorials/08-cairo-1-support/03-cheatcodes/deploy_contract_cairo.md

```diff
@@ -0,0 +1,35 @@
+# `deploy_contract_cairo0`
```

Comment:
why is the `0` missing from the file name?

Reply:
https://github.com/software-mansion/protostar/pull/1840/commits/9984117aeeb36097db629c0af69223c212c4cb91